src/tests/pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[tool.jupysql.SqlMagic]
+autocom = true
+autolimit = "text"
+invalid = false
```

Comment:
this should be in a `tests/assets` folder if at all needed. You can also generate this file in the test itself.
You can also add test cases with different folder structures like in [pkgmt](https://github.com/ploomber/pkgmt/tree/main/tests/assets)

Reply:
agree with @neelasha23, it'd be better to generate this in the tests. you can use the `tmp_empty` fixture to guarantee that your test will run in a tmp directory unique to your test

```python
from pathlib import Path

def test_something(tmp_empty):
    Path("pyproject.toml").write_text("""
[tool.jupysql.SqlMagic]
autocom = true
autolimit = "text"
invalid = false
""")
```

Reply:
Now creating content of `pyproject.toml` files with `pytest.mark.parametrize` under tmp directory using `tmp_empty`.